packages/deployment/app/cloudformation.template.yml

```diff
@@ -155,7 +155,7 @@ Resources:
       {{else}}
       Endpoint: {{this.endpoint}}
       {{/if}}
-      Protocol: {{@key}}
+      Protocol: lambda
```

Comment:
to make things more flexible in the future, could you add a key:value called `protocol: lambda` to the config and reference it here as `{{this.protocol}}`?

Reply:
Yeah good idea